src/b-splines/constant.jl

```diff
@@ -1,20 +1,50 @@
 immutable Constant <: Degree{0} end
 
+"""
+Constant b-splines are *nearest-neighbor* interpolations, and effectively
+return `A[round(Int,x)]` when interpolating
+
+Also, although the implementation is slightly different in order to re-use
```

Comment:
This is a copy-paste mistake; this remark is about linear, not constant, B-splines.